contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -29,6 +29,10 @@ type htlcIncomingContestResolver struct {
 	// circuitKey describes the incoming htlc that is being resolved.
 	circuitKey channeldb.CircuitKey
 
+	// payload is the hop payload which can contain additional data that is
+	// required for resolution.
+	payload *hop.Payload
```

Comment:
make `invoices.Payload`?

Reply:
do you suggest renaming the whole package?

Reply:
I’m suggesting the type be the interface `invoices.Payload` instead of the concrete type